uber/config.py

```diff
@@ -375,8 +375,10 @@ def _is_intstr(s):
 c.CON_LENGTH = int((c.ESCHATON - c.EPOCH).total_seconds() // 3600)
 c.START_TIME_OPTS = [(dt, dt.strftime('%I %p %a')) for dt in (c.EPOCH + timedelta(hours=i) for i in range(c.CON_LENGTH))]
 c.DURATION_OPTS = [(i, '%i hour%s' % (i, ('s' if i > 1 else ''))) for i in range(1, 9)]
-c.SETUP_TIME_OPTS = [(dt, dt.strftime('%I %p %a')) for dt in (c.EPOCH - timedelta(days=5) + timedelta(hours=i) for i in range(16))] \
-                  + [(dt, dt.strftime('%I %p %a')) for dt in (c.EPOCH - timedelta(days=1) + timedelta(hours=i) for i in range(24))]
+c.SETUP_TIME_OPTS = [(dt, dt.strftime('%I %p %a'))
+                     for dt in (c.EPOCH - timedelta(days=day) + timedelta(hours=hour)
+                                for day in range(c.SETUP_SHIFT_DAYS, 0, -1)
+                                for hour in range(24))]
```

Comment:
``range()`` can go up or down, and by a specifiable step.  Not often used, but really useful in cases like this.